nautobot/core/tests/test_api.py

```diff
@@ -281,12 +281,28 @@ def test_get_lookup_field_data_invalid_query_params(self):
 
     def test_get_lookup_value_dom_element(self):
         url = reverse("core-api:filtersetfield-retrieve-lookupvaluedomelement")
-        response = self.client.get(url + "?content_type=dcim.site&field_name=name", **self.header)
-
-        self.assertEqual(response.status_code, 200)
-        self.assertEqual(
-            response.data,
-            {
-                "dom_element": '<select name="name" class="form-control nautobot-select2-multi-value-char" data-multiple="1" id="id_for_name" multiple>\n</select>'
-            },
-        )
+        with self.subTest("Assert correct lookup field dom element is generated"):
+            response = self.client.get(url + "?content_type=dcim.site&field_name=name", **self.header)
```

Comment:
Because Site goes away in `next` can we move away from it being a test example? In this case we don't need any additional data so maybe something like Location or Manufacturer?